packages/block-editor/src/components/block-edit/edit.js

```diff
@@ -29,7 +29,25 @@ import BlockContext from '../block-context';
  */
 const DEFAULT_BLOCK_CONTEXT = {};
 
-export const Edit = ( props ) => {
+const Edit = ( props ) => {
+	const { name } = props;
+	const blockType = getBlockType( name );
+
+	if ( ! blockType ) {
+		return null;
+	}
+
+	// `edit` and `save` are functions or components describing the markup
+	// with which a block is displayed. If `blockType` is valid, assign
+	// them preferentially as the render value for the block.
+	const Component = blockType.edit || blockType.save;
+
+	return <Component { ...props } />;
+};
+
+const EditWithFilters = withFilters( 'editor.BlockEdit' )( Edit );
```

Comment:
Pinging @youknowriad to double check that it wasn't intentional that we don't pass `context` and `className` to components that use `editor.BlockEdit`.

I would expect that components using the `editor.BlockEdit` filter receive the exact same props as a block's `edit` component, which is what this change results in.

Reply:
Cool yeah me neither thanks for the gut check!

Reply:
Looks like this might have caused a breaking change for folks relying on the `editor.BlockEdit` filter to pass context to BlockEdit; see report in #53869. I'm not sure if that was intended behaviour to begin with 😅 but applying the filters to the internal Edit component instead of the outer wrapper results in it no longer being possible.